### PR TITLE
Setup initial starter kit

### PR DESCRIPTION
**Pull request by bertrandk**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const webpackHotMiddleware = require('webpack-hot-middleware');
+const config = require('./webpack.config');
+
+
+const app = express();
+const PORT = process.env.PORT || 3000;
+const compiler = webpack(config);
+
+
+app.use(webpackDevMiddleware(compiler, {
```
Comment (SethDavenport) [inline server.js]: any reason you're not going with `serve-webpack-client` like the other starters?

**Comment by SethDavenport** [comment]
Otherwise, LGTM

File: server.js (hunk 1 above)
Comment (bertrandk) [inline server.js]: I needed the generic one to debug error codes...will work `serve-webpack-client` back in.

Comment (SethDavenport) [inline server.js]: OK makes sense

**Pull request merged**